### PR TITLE
[CodeCamp2023-495]Translation into Chinese of an English document.

### PR DESCRIPTION
**Pull request by yeungkong**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

I would like to translate an English document into Chinese. I have created a pull request before. But it was failed. This is the second time that I create the pull request.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMPreTrain.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/zh_cn/user_guides/config2.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# 学习更多与配置相关的事
```
Comment (hhaAndroid) [inline docs/zh_cn/user_guides/config2.md:1]: docs/zh_cn/user_guides/config2.md -> docs/zh_cn/user_guides/tracking_config.md

File: docs/zh_cn/user_guides/config2.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+我们用python文档作为我们的配置系统。你可以在`MMDetection/configs`底下找到所有已提供的配置文件。
```
Comment (hhaAndroid) [inline docs/zh_cn/user_guides/config2.md:3]:
```suggestion
我们用 python 文档作为我们的配置系统。你可以在 `MMDetection/configs` 底下找到所有已提供的配置文件。
```

Comment (hhaAndroid) [reply docs/zh_cn/user_guides/config2.md:3]: 所有中英文和数字之间都需要空格

File: docs/zh_cn/user_guides/config2.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+我们把模块化和继承化设计融入我们的配置系统，这使我们很方便去进行各种实验。如果你想查看相关的配置文件，你可以跑`python tools/misc/print_config.py/PATH/TO/CONFIG`去看完整的详细配置。
```
Comment (hhaAndroid) [inline docs/zh_cn/user_guides/config2.md:5]:
```suggestion
我们把模块化和继承化设计融入我们的配置系统，这使我们很方便去进行各种实验。如果你想查看相关的配置文件，你可以跑 `python tools/misc/print_config.py/PATH/TO/CONFIG` 去看完整的详细配置。
```

File: docs/zh_cn/user_guides/config2.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+## 完整配置的简要说明
+
+一个完整的配置通常包含以下主要的字段：
+
+`model`：一个模型的基本配置，包含`data_preprocessor`、`detector`、`motion`之类的模块，还有`train_cfg`、`test_cfg`等等；
```
Comment (hhaAndroid) [inline docs/zh_cn/user_guides/config2.md:11]:
```suggestion
`model`：一个模型的基本配置，包含 `data_preprocessor`、`detector`、`motion` 之类的模块，还有 `train_cfg`、`test_cfg` 等等；
```

Comment (hhaAndroid) [reply docs/zh_cn/user_guides/config2.md:11]: 后面的类似空格问题，麻烦都改一下

**Comment by hhaAndroid** [comment]
@yeungkong Please sign the CLA

**Comment by yeungkong** [comment]
Hi @hhaAndroid. I have made the changes according to your request. Please check that.

File: docs/zh_cn/user_guides/tracking_config.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
+# 学习更多与配置相关的事
+
+我们用 python 文档作为我们的配置系统。你可以在 `MMDetection/configs` 底下找到所有已提供的配置文件。
+
+我们把模块化和继承化设计融入我们的配置系统，这使我们很方便去进行各种实验。如果你想查看相关的配置文件，你可以跑 `python tools/misc/print_config.py/PATH/TO/CONFIG` 去看完整的详细配置。
```
Comment (hhaAndroid) [inline docs/zh_cn/user_guides/tracking_config.md:5]:
```suggestion
我们把模块化和继承化设计融入我们的配置系统，这使我们很方便去进行各种实验。如果你想查看相关的配置文件，你可以跑 `python tools/misc/print_config.py /PATH/TO/CONFIG` 去看完整的详细配置。
```

**Pull request merged**